src/App.js

```diff
@@ -0,0 +1,32 @@
+import "./App.css";
+import { BrowserRouter, Redirect, Route } from "react-router-dom";
+import CountryCardsContainer from "./component/CountryCards/CountryCardsContainer";
+import InputSearchContainer from "./component/Input/InputSearchContainer";
+import CountryPageContainer from "./component/CountryPage/CountryPageContainer";
+
+const App = (props) => {
+  return (
+    <div className="App">
+      <BrowserRouter>
+        <Route
+          path="/card/:nameCountry"
+          render={() => <CountryPageContainer />}
+        />
+        <Route
+          path="/country"
+          render={() => (
+            <div>
+              <InputSearchContainer />
+              <CountryCardsContainer />
```

Comment:
it is better to group this two component into the page component and add page component here